keep/contextmanager/contextmanager.py

```diff
@@ -45,11 +45,12 @@ def __init__(self):
 
         self.state_file = self.STATE_FILE or os.environ.get("KEEP_STATE_FILE")
         self.steps_context = {}
+        self.actions_context = {}
         self.providers_context = {}
         self.alert_context = {}
-        self.foreach_context = {}
-        self.foreach_conditions = []
-        self.foreach_index = None
+        self.foreach_context = {
+            "value": None,
+        }
```

Comment:
?

Reply:
this is to support `foreach.value` and remove the hack that `get_full_context` had.